src/test/kotlin/com/michaelsgroi/baseballdatabank/BaseballDatabankTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.michaelsgroi.baseballdatabank
+
+import com.michaelsgroi.baseballreference.Career.Companion.relevantAwards
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class BaseballDatabankTest {
+
+    @Test
+    fun test() {
```

Comment:
more tests?

Reply:
sorry that probably sounded almost snarkily vague: more tests in particular to at least drive an explanation of how the new code should work (not quite TDD, but what the end(ish)game of TDD should look like I guess)